docs/low-code-spark/pubsub/pubsub.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Publish-Subscribe Data Projects
+description: Publish-Subscribe Data Projects
+id: pubsub
+tags: []
+---
+
+One of the best practices during Data-Project development is to _Use standardized components_. Prophecy enables this standardization in a few ways:
+
+1. Datasets
+2. User-Defined Functions
+3. Configured Subgraphs
+4. Configured Pipelines
+
+`Pipeline`s, `Dataset`s, `Subgraph`s, and `User-Defined Function`s can now be shared across multiple projects and teams. This allows central Data Platform teams to build reusable code to cover a wide variety of business needs, such as Encryption/Decryption or Identity Masking, and have their "consumers" (the Data Practitioners) take a dependency on that reusable code. Since it's all versioned, when the reusable code changes, the downstream consumers will be notified and can update accordingly.
+
+Data admins can also _Create deployment templates for the pipelines_ that have the best practices baked into them for authorization, notifications,
+handling of errors, and logging the correct information.
+
+Prophecy allows the Data Platform teams to create and **Publish** these standards to the various teams.
+
+To share these across projects, they would need to release the Project which has these Pipelines/UDFs defined.
+
+Please refer [here](../../metadata/Git#how-to-release-a-branch) for how to release a Project.
+
```

Comment:
For this page can we create a fake Project (like you've been doing in other pages, _Base Project_ for example) and use that across all of these sections? The user would be able to see adding _Base Project_ as a dependency, see the components of it in the new/existing project, then updating the version.